### ISSUE TITLE
react-native-webrtc - typings missing support for datachannels

### ISSUE THREAD

**Issue by eggysplat** [DefinitelyTyped/DefinitelyTyped#45818]
- [x] I tried using the `@types/react-native-webrtc` package and had problems.
- [x] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [x] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @KarlosQ 

If you do not mention the authors the issue will be ignored.

So, I have tried to use react-native-webrtc (v1.75.3) with the latest typings library (1.75.1) and there is no typings for datachannels.  Obviously, typescript transpiles back down to javascript so I can get the project to run as intended but I've had to use an awful lot of //@ts-ignore lines in order for webpack to compile the project.

Things that I have noticed that have caused quite a lot of pain:
- `new MediaStream` - Typings forces us to use an argument. afaik, this argument is optional, and the project runs fine when no argument is supplied.
- `RTCPeerConnection.createDataChannel` returns `Promise<void>` in the typings file, whereas the actual return type is `Promise<RTCDataChannel>`
- `RTCDataChannel` is not defined in the typings file
- `RTCPeerConnection.ondatachannel` is not defined in the typings file
- `getUserMedia` may be imported from `react-native-webrtc/getUserMedia` (default import). Typings has no support for this.

There may well be others. These are the ones that I have had to //@ts-ignore in the project I am working on though

**Comment by salmazov** [comment]
I agree. createDataChannel doesn't return void / Promise<void> but RTCDataChannel. Should be fixed in master
